src/yaml/enums.cr

```diff
@@ -0,0 +1,36 @@
+module YAML
```

Comment:
Why not leave 'em in `LibYAML` and use `alias` instead of moving all the enums?

Reply:
Because API docs don't show for `lib` types.